015-customer-io-transactionals.md

```diff
@@ -0,0 +1,44 @@
+In 2021, we began to [only maintain Customer.io profiles for active members who have opted in to receive email and/or SMS promotions](https://www.pivotaltracker.com/epic/show/4721712). The [workflows we use in Customer.io](https://customer.io/visual-workflow-builder/) are convenient because they provide a no-code way to send transactional email and/or SMS messages -- but it does require that a member has a Customer.io profile in order to receive the transactional message. It's also a bit brittle (e.g. -- an admin could accidentally delete the workflow that sends our SMS compliance message if someone opts-in to receive texts via web)
```

Comment:
The ability to accidentally disable our SMS compliance message definitely seems like a big risk factor & a good argument for moving critical messaging flows like this into code. 👍